face_recognition/__init__.py,sha256=CM-6RP2294Es5UsgmYAxNk1ki2HCa88IYOs2UMmANJw,224
face_recognition/api.py,sha256=MzU1jHSN2A1KXqa2Hj_zCAOwttgqBhebLt8lbEyUkbE,6762
face_recognition/cli.py,sha256=_7OnVt7UovT8OEDMcOYp95UM6xFKoTnQAoim-H2l1CQ,2468
face_recognition-0.1.14.dist-info/DESCRIPTION.rst,sha256=fKS0vLc3Exu1qXjXV_M4BIcbima375O00Nm2dHh7ZaU,13828
face_recognition-0.1.14.dist-info/METADATA,sha256=rCVRp41kHlrFv-RLu2aWxCJnakO-HWzykKSikajjO7U,14876
face_recognition-0.1.14.dist-info/RECORD,,
face_recognition-0.1.14.dist-info/WHEEL,sha256=o2k-Qa-RMNIJmUdIc7KU6VWR_ErNRbWNlxDIpl7lm34,110
face_recognition-0.1.14.dist-info/entry_points.txt,sha256=_E-C4Ey17oCQWPeU_4u-3O2yqmrLaWTHrykvpzlq88I,64
face_recognition-0.1.14.dist-info/metadata.json,sha256=l0E2HzOJU7Qt-v7xy_isoMxOzmtqHu4BNbFhiidKN3g,1356
face_recognition-0.1.14.dist-info/top_level.txt,sha256=ExXWSKN18HXPKa1CC6ev3GV-Be75gpzSrm4bZElYfbQ,17
